manifests/init.pp

```diff
@@ -175,24 +177,19 @@
     validate_re($logdir,'^(\$(?!/)|/).+')
     validate_re($rundir,'^(\$(?!/)|/).+')
 
+    if $ca_crl_pull_interval != 2 {
+      deprecation('pupmod::ca_crl_pull_interval', 'pupmod::ca_crl_pull_interval is deprecated, the CRL cron job has been removed.')
```

Comment:
@jeefberkey Are you OK with just ignoring the option completely and simply adding to the comments? There's no reason to bother people about this.